### DISCUSSION TITLE
Language localization detect & config not valid

### DISCUSSION THREAD

**Discussion by KVM-Explorer** [jellyfin/jellyfin-web#6093] [Q&A]
- My Jellyfin Verison & Web Client 10.9.8
-  Windows 10 

I‘m using Jellyfin web client，when I use another device or log out for a long time，I found my langunage change to default english. Even though I set Chinese in user's localization config, it will change to default "Auto". For all of these I have two questions. 

Where to store relevant configurations?
 How to config can solve my problem?

**Comment by thornbill** [comment]
Please refer to [this](https://jellyfin.org/docs/general/getting-help) to find the appropriate places for help troubleshooting.
